packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -2,17 +2,19 @@
 import '../../solid-components';
 import './quickfact';
 import { html } from 'lit-html';
-import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
-import cx from 'classix';
 
 const { argTypes, parameters } = storybookDefaults('sd-quickfact');
 const { overrideArgs } = storybookHelpers('sd-quickfact');
 const { generateTemplate } = storybookTemplate('sd-quickfact');
 
 /**
+ *
+ *  Used to display a quick fact item with an optional summary section.
+ *
+ *  **Related templates:**
```

Comment:
Please use the same quickfact sample as in figma

Reply:
Updated